### PR TITLE
demo/polar-btc-chart

### PR DESCRIPTION
**Pull request by bm64**
Added polar bitcoin demo.

### CODE REVIEW

**Review by DJTechnoo — COMMENTED** [review]
Looks great!

**Review by DJTechnoo — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]
Nothing to add

**Review by pawelfus — APPROVED** [review]
Remaining questions:
- this demo could also be available under Highcharts categories, are you okay @TorsteinHonsi with placing it in the Stock-related category?
- reported two bugs, related to the background: https://github.com/highcharts/highcharts/issues/20743 and https://github.com/highcharts/highcharts/issues/20744 - I think plotBand workaround works better. Perhaps we should set a fixed max?

**Review by kamil-musialowski — APPROVED** [review]
Demo and code are great, just one hint.

If we hide the 2021 series (by legend click), then the dark blue background is hidden and the white labels are not visible. Can we make this background visible on both series?

**Review by marvin19 — CHANGES_REQUESTED** [review]
Really cool demo! 🤩 A few requests:
- Add `xAxis.accessibility.description` to a suitable name for the xAxis
- Add `yAxis.accessibility.description `to a suitable name for the yAxis
- Change the 0 to black font color or another color with more than 4.5:1 contrast to the yellow. It is good enough for 25k and 50k 👌🏻

**Review by marvin19 — APPROVED** [review]
Looks good!

File: samples/highcharts/blog/polar-btc/demo.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
```
Comment (pawelfus) [inline samples/highcharts/blog/polar-btc/demo.html:3]: Sorry for the late nitpicking, but could you keep all modules to be loaded for HC? I mean, remove `stock` from URL

**Review by pawelfus — CHANGES_REQUESTED** [review]
Just one tiny thing

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! There are some lint errors now with the stricter lint rules.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**